src/coffee/extend/ForceDirectedGraph.coffee

```diff
@@ -0,0 +1,154 @@
+Function::property = (prop, desc) ->
+  Object.defineProperty @prototype, prop, desc
+
+
+# Hacky Edge implementation
+class Edge
+  constructor: (source, target) ->
```

Comment:
Instead of a new class with `source` and `target`, I wonder if this can be just a `Pair` or `Line` 

eg: `var edge = new Line( source ).to( target )` ?